fixity/fixity.py

```diff
@@ -305,7 +309,9 @@ def get_handler(stream, timestamps, log_level=None):
     return handler
 
 
-def main(argv=None, logger=None, stream=None):
+def main(
+    argv: List[str] = None, logger: logging.Logger = None, stream: TextIO = None
```

Comment:
I think all these parameters that default to `None` should be marked as [Optional](https://docs.python.org/3.8/library/typing.html#typing.Optional).